index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+  
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="portfolio.css">
+
+</head>
+<body>
+    <!-- TOOLBAR -->
+    <header class="toolbar">
+        <div class="logo"> Welcome </div>
+        <nav class="navbar"> <img src="images/icons/menu.png" alt="menu"> </nav>
+    </header>
+    <!-- END TOOLBAR -->
+
+    <!-- HOME -->
+    <section class="home">
+        <h2 class="home_title">
+            <span>Hey there. I’m Jihane</span>
+            <span class="home_title_green">I’m a software developer </span>
+        </h2>
+        <p class="home_text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+        <div class="home_socialbar">
+            <img src="images/icons/twitter.png" alt="twitter link">
+            <img src="images/icons/linkedin.png" alt="linkedin link">
+            <img src="images/icons/medium.png" alt="medium link">
+            <img src="images/icons/github.png" alt="github link">
+            <img src="images/icons/angellist.png" alt="angellist link">
+        </div>    
```

Comment:
- [ ] This `<div>` element has been rendered well, however this can be represented in a better semantic way - _(Remember writing semantic HTML is a good practise)_. I am suggesting that the list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag).

Reply:
Done, thank you